src/main/java/net/ripe/ipresource/Ipv4Address.java

```diff
@@ -177,7 +171,7 @@ public int getCommonPrefixLength(UniqueIpResource other) {
 
     @Override
     public Ipv4Address lowerBoundForPrefix(int prefixLength) {
-        long mask = ~((1L << (NUMBER_OF_BITS - prefixLength)) -  1);
+        long mask = -(1L << (NUMBER_OF_BITS - prefixLength));
```

Comment:
There's some consistency in this class on how `1L << NUMBER_OF_BITS` is used. 
So I would either not have this change at all or have it replaces in all places.

Reply:
Can you add a test to cover this case to be sure the behavior remains? For example if prefix length is 8, these two versions produce different long value.

```scala
@ (1L << 24) - 1
res0: Long = 16777215L

@ res0.toBinaryString
res1: String = "111111111111111111111111"

@ -(1L << 24)
res2: Long = -16777216L

@ res2.toBinaryString
res3: String = "1111111111111111111111111111111111111111000000000000000000000000"

```

Reply:
It seems I forgot to type ~ on the first line. So it's fine, ignore my comment :)

Reply:
I do not like changes in logic that uses bitwise operations (and possibly the number system of the hardware).

This specific one is identical (https://colab.research.google.com/drive/1cH2_a3Cxww7iqUwFd26OSsC2pkYjqDWI?usp=sharing for proof).

Reply:
Since there is no consensus I reverted the change.